juriscraper/opinions/united_states/federal_special/uscgcca.py

```diff
@@ -0,0 +1,76 @@
+# Scraper for US Coast Guard Court of Criminal Appeals
+
+from juriscraper.lib.string_utils import convert_date_string
+from juriscraper.OpinionSite import OpinionSite
+from re import search
+from requests.utils import requote_uri
+
+
+class Site(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(Site, self).__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.mj_cite_regex = "\\d+ *M\\.*J\\.* *\\d+"
+        self.wl_cite_regex = "\\d+ *W\\.*L\\.* *\\d+"
+        self.current_page = 1
+        self.base_url = (
+            "https://www.uscg.mil/Resources/Legal/"
+            "Court-of-Criminal-Appeals/"
+            "CGCCA-Opinions/smdpage15701/{}"
+        )
+        self._format_url()
+
+    def _format_url(self):
+        self.url = self.base_url.format(self.current_page)
+
+    def _process_html(self):
+        table_xpath = "//table[@class='Dashboard']//tbody[1]"
+        self.html = self.html.xpath(table_xpath)[0]
+
+    def _get_case_dates(self):
+        dates_xpath = "//tr//td[3]"
+        return [
+            convert_date_string(date.text_content())
+            for date in self.html.xpath(dates_xpath)
+        ]
+
+    def _get_case_names(self):
+        names_xpath = "//tr//td[1]"
+        names_regex = "^([a-zA-Z ]+ [Vv]\\.? [ ]*[a-zA-Z]+)"
```

Comment:
This example from the HTML 

`ORDER - In re Tucker Petition 2019 WL 5280525`

Should extract `In re Tucker Petition`

This is a hard thing about regex and legal stuff.